packages/cli/src/databases/types.ts

```diff
@@ -1,5 +1,5 @@
 import type { INodeTypes } from 'n8n-workflow';
-import type { QueryRunner, ObjectLiteral } from 'typeorm';
+import { type QueryRunner, type ObjectLiteral } from 'typeorm';
```

Comment:
Is this needed? Seems all the other type imports use the former syntax

Reply:
Autofix being autofix :)